Adyen/UI/Form/Items/Text/FormTextItemView.swift

```diff
@@ -316,6 +323,12 @@ open class FormTextItemView<ItemType: FormTextItem>: FormValueItemView<String, F
         }
     }
     
+    private func resetValidationStatus() {
+        accessory = .none
+        hideAlertLabel(true)
+        highlightSeparatorView(color: tintColor)
```

Comment:
this should be:

```
highlightSeparatorView(color: item.style.separatorColor)
```
tintColor is when its highlighted, right? but since we're reseting its shouldn't be highlighted, am I missing something?

Reply:
Actually, you're right. I should've used `defaultSeparatorColor` instead to keep the separator view with a gray color.